save-backend/build.gradle.kts

```diff
@@ -4,6 +4,13 @@ import  org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 plugins {
     kotlin("jvm")
     id("org.jetbrains.kotlin.plugin.jpa") version Versions.kotlin
+    id("org.jetbrains.kotlin.plugin.allopen") version Versions.kotlin
```

Comment:
I believe there is shortcut for official plugins, like `kotlin("plugin.allopen")`